src/TestProblem/Hydro/CCSN/dtSolver_CCSN.cpp

```diff
@@ -98,35 +104,31 @@ double Mis_GetTimeStep_Lightbulb( const int lv, const double dTime_dt )
                real *B = NULL;
 #              endif
 
+               SrcTerms.Lightbulb_CPUPtr( fluid, B, &SrcTerms, 0.0, NULL_REAL, x, y, z, NULL_REAL, NULL_REAL,
```

Comment:
I agree that we should have `dt = 0.0`. Just wondering whether it was a bug when assigning `dt=NULL_REAL` even when `GAMER_DEBUG` is disabled?

Reply:
When disabling `GAMER_DEBUG`, I think assigning `dt=NULL_REAL` will not trigger any bug. In fact, it only print out more (false-positive) error messages when enabling `GAMER_DEBUG`.